aws/deploy.sh

```
@@ -0,0 +1 @@
+aws cloudformation deploy --template-file aws/templates/api-gateway-ses.yml --stack-name uji52com-dev-apigw-ses --parameter-overrides EmailSNSTopicArn=arn:aws:sns:ap-northeast-1:xxxxxxxx:SendEmail --capabilities CAPABILITY_IAM
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)